src/ethereum_spec_tools/lint/__init__.py

```diff
@@ -11,11 +11,45 @@
 import pkgutil
 from abc import ABCMeta, abstractmethod
 from dataclasses import dataclass
+from itertools import zip_longest
 from typing import Generator, List, Optional, Sequence, Tuple
 
 from ..forks import Hardfork
 
 
+def compare_ast(old: ast.AST, new: ast.AST) -> bool:
+    """
+    Check if two nodes are the equal.
+    """
+    if type(old) is not type(new):
+        return False
+
+    if isinstance(old, ast.Expr):
+        return True
```

Comment:
As discussed on call, we should probably limit this check to expressions that contain only a single string and document that we're ignoring docstrings.

Reply:
done